sdk/vision/azure-ai-vision-imageanalysis/src/main/java/com/azure/ai/vision/imageanalysis/ImageAnalysisOptionsBuilder.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.ai.vision.imageanalysis;
+
+import java.util.List;
+
+/**
+ *  Builder class to set options for a single Image Analysis operation.
+ */
+public final class ImageAnalysisOptionsBuilder {
```

Comment:
Was there a specific reason to have the Options type have a builder? Generally, Options types are fluent and not immutable.

Reply:
After further discussion we agreed to remove the ImageAnalysisOptionsBuilder class. I will submit a commit to this PR that will update ImageAnalysisOptions with new "setXxx" methods that support chained calls. I will leave this comment open until the commit is done.

Reply:
Resolving after latest commit that removed ImageAnalysisOptionsBuilder. Thanks @alzimmermsft!